scripts/question_answering/README.md

```diff
@@ -0,0 +1,113 @@
+*   [evaluate-v2.0.py](https://worksheets.codalab.org/rest/bundles/0x6b567e1cf2e041ec80d7098f031c5c9e/contents/blob/)
+
+download and move them under `$DATA_DIR`
+
+### Running Script
+We provide the script to train on the SQuAD dataset.
+
+```bash
+VERSION=2.0  # Either 2.0 or 1.1
+MODEL_NAME=google_albert_base_v2
+
+# Prepare the Data
+nlp_data prepare_squad --version ${VERSION} 
+
+# Run the script
+python run_squad.py \
+    --model_name ${MODEL_NAME} \
+    --data_dir squad \
+    --output_dir fintune_${MODEL_NAME}_squad_${VERSION} \
+    --version ${VERSION} \
+    --do_eval \
+    --do_train \
+    --batch_size 4 \
+    --num_accumulated 3 \
+    --gpus 0,1,2,3 \
+    --epochs 3 \
+    --lr 2e-5 \
+    --warmup_ratio 0.1 \
+    --wd=0.01 \
+    --max_seq_length 512 \
+    --max_grad_norm 0.1 \
+    --overwrite_cache \
+```
+or evaluate SQuAD1.1 based on a SQuAD2.0 fine-tuned checkpoint as
+
+```bash
+python run_squad.py \
+    --model_name ${MODEL_NAME} \
+    --data_dir=squad \
+    --output_dir=${OUT_DIR} \
+    --param_checkpoint=${CKPT_PATH} \
+    --version 2.0 \
+    --do_eval \
+    --gpus 0,1,2,3,4,5,6,7 \
+    --eval_batch_size 16 \
+    --overwrite_cache \
+```
+
+### Results
+We reproduced the ALBERT model which is released by Google, and fine-tune the the SQuAD with single models. ALBERT Version 2 are pre-trained without the dropout mechanism but with extra training steps compared to the version 1 (see the [original paper](https://arxiv.org/abs/1909.11942) for details).
+
+Fine-tuning the listed models with hyper-parameter learning rate 2e-5, epochs 3, warmup ratio 0.1 and max gradient norm 0.1 (as shown in command). Notice that the `batch_size` is set for each GPU and the global batch size is 48 for all experiments, besides that gradient accumulation (`num_accumulated`) is supported in the case of out of memory.
+
+Performance are shown in the table below, in which the SQuAD1.1 are evaluated with SQuAD2.0 checkpoints.
+
+|Reproduced ALBERT Models  | SQuAD 1.1 dev  | SQuAD 2.0 dev |
+|--------------------------|---------------|--------------|
+|ALBERT base               | 90.55/83.83   | 82.13/79.19  |
+|ALBERT large              | 92.66/86.43   | 84.93/82.02  |
+|ALBERT xlarge             | 93.85/87.71   | 87.61/84.59  |
+|ALBERT xxlarge            | 95.00/89.01   | 90.0/86.9    |
+
+For reference, we've included the results from Google's Original Experiments
+
+| Model Name | SQuAD 1.1 dev | SQuAD 2.0 dev|
+|------------|---------------|--------------|
+|ALBERT base (googleresearch/albert)    | 90.2/83.2     | 82.1/79.3    |
+|ALBERT large (googleresearch/albert)   | 91.8/85.2     | 84.9/81.8    |
+|ALBERT xlarge (googleresearch/albert)  | 92.9/86.4     | 87.9/84.1    |
+|ALBERT xxlarge (googleresearch/albert) | 94.6/89.1     | 89.8/86.9    |
+
+For BERT and ELECTRA model, the SQuAD 1.1 is evaluated with SQuAD 2.0 checkpoints, as the following results showcased.
```

Comment:
Actually, for BERT and ELECTRA model, the SQuAD 1.1 is evaluated with SQuAD 1.1 itself.